common/djangoapps/student/tests/test_models.py

```diff
@@ -244,6 +256,169 @@ def test_enrollments_not_deleted(self):
         self.assertEqual(enrollment_refetched.all()[0], enrollment)
 
 
+@override_waffle_flag(REDIRECT_TO_COURSEWARE_MICROFRONTEND, active=True)
+@override_waffle_flag(COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES, active=True)
+@override_waffle_flag(COURSEWARE_MICROFRONTEND_PROGRESS_MILESTONES_STREAK_CELEBRATION, active=True)
+class UserCelebrationTests(SharedModuleStoreTestCase):
+    """
+    Tests for User Celebrations like the streak celebration
+    """
+    @classmethod
+    def setUpClass(cls):
+        super(UserCelebrationTests, cls).setUpClass()
+        cls.course = CourseFactory()
+        cls.course_key = cls.course.id  # pylint: disable=no-member
+
+    def setUp(self):
+        super(UserCelebrationTests, self).setUp()  # lint-amnesty, pylint: disable=super-with-arguments
```

Comment:
New code - just drop the pylint disable and modernize the code: `super().setUp()`